.gitignore

```diff
@@ -37,3 +37,10 @@ src
 .vscode
 
 vignettes/results
+
+inst/stan/dist_fit
```

Comment:
It might be marginally safer to `.gitignore` all of `inst/stan` and then allow just `*.stan`. Pretty minimal though